### ISSUE TITLE
more updating for datastorr

### ISSUE THREAD

**Issue by wcornwell** [wcornwell/taxonlookup#18]
```
taxonlookup:::plant_lookup_info()
Error in datastorr::github_release_info("traitecoevo/taxonlookup", filename = "plant_lookup.csv",  : 
  argument "path" is missing, with no default
```

ping: @richfitz 

maybe:

```
plant_lookup_version_current <- function(local=TRUE, path=NULL) {
  datastorr::github_release_version_current(plant_lookup_info(info), local)
}
```

should be

```
plant_lookup_version_current <- function(local=TRUE, path=NULL) {
  datastorr::github_release_version_current(plant_lookup_info(path), local)
}
```

**Comment by richfitz** [comment]
yeah, that looks about right.

This does mean there's no test though so I'll add one at the same time.

**Comment by wcornwell** [comment]
If you're going to work on tests, check:

https://codecov.io/github/traitecoevo/taxonlookup/R/plant_lookup.R?ref=d73b95f9fe89aef569b236fd00733b8eeaeca497

and 

https://codecov.io/github/traitecoevo/taxonlookup/R/utils.R?ref=d73b95f9fe89aef569b236fd00733b8eeaeca497

**Comment by wcornwell** [comment]
fixed here: 122d7fe12081ae91757c9ff10c34c951cb552f5d
